README.md

```diff
@@ -45,3 +45,7 @@ In production, deploy and upload your plugin via the [System Console](https://ab
 ### How do I make a server-only or web app-only plugin?
 
 Simply delete the `server` or `webapp` folders and remove the corresponding sections from `plugin.json`. The build scripts will skip the missing portions automatically.
+
+### How do I include assets in the plugin bundle?
+
+Place them into `assets` subfolder. You can change this directory name by editing `ASSETSDIR` in `Makefile`.
```

Comment:
It may not be obvious exactly where this `assets` directory must live -- I could see someone thinking it belongs in `server` or `webapp`. Might we consider either pre-creating this folder and adding a `.gitignore` to it so that it actually gets checked out?

Separately, I feel like teaching users how to override `ASSETS_DIR` at all is overkill: no objection to actually /making/ it configurable, but the use case for renaming this folder seems low and the benefit to simplifying the description much higher. I wonder if this "space" would be more useful describing how to use the assets:

> Place them into the `assets` directory. To use an asset at runtime, build the path to your asset and open as a regular file:
```
assetsPath, err := p.API.GetAssetsPath();
if err != nil {
    return errors.Wrap(err, "failed to get assets path")
}

profileImage, err := ioutil.ReadFile(filepath.Join(assetsPath, "profile_image.png"))
if err != nil {
    return errors.Wrap(err, "failed to read profile image")
}

appErr := p.API.SetProfileImage(userId, filepath.Join(assetsPath, "profile_image.png"))
if appErr != nil {
    return errors.Wrap(err, "failed to set profile image")
}
```

Reply:
:+1: for both suggestions. I added a `.gitkeep` into `assets` instead of  `.gitignore`, but I'm only 1/5 on the file name.